src/components/linkcarousel/LinkCarousel.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Carousel } from 'antd';
+import styled from 'styled-components';
+import { LIGHT_GREEN } from '../../colors';
+import LinkCard from '../linkcard/LinkCard';
+import useWindowDimensions, { WindowTypes } from '../window-dimensions';
+
+const StyledCarousel = styled(Carousel)`
+  max-width: 1350px;
+  margin: auto;
+  border: solid;
+  border-radius: 10px;
+  border-width: 5px 5px 5px 5px;
+  border-color: ${LIGHT_GREEN};
+`;
+
+const CarouselSlide = styled.div`
+  padding: 10px;
+  border: solid;
+  border-width: 0px 0px 50px 0px;
+  border-color: ${LIGHT_GREEN};
+  text-align: center;
+`;
+
+const LinkCarousel: React.FC = () => {
+  const { windowType } = useWindowDimensions();
+  const slidesPerPage = Number(
+    `${
+      windowType === WindowTypes.Desktop
+        ? 4
+        : `${
+            windowType === WindowTypes.NarrowDesktop
+              ? 3
+              : `${windowType === WindowTypes.Mobile ? 1 : 2}`
```

Comment:
So this is saying if it's mobile show 1, otherwise show 2? Is the other option Tablet? If so, could you add another check for tablets for clarity, or a comment will do too, just so for code readability. If the other option isn't Tablet then you can leave it this way

Reply:
Right, there's only four options for `windowType` so checking the other three should mean it's Tablet. That assumption might be a problem if we add another window type - I could change that mobile check to show 2 if tablet, otherwise show 1 since 1 is for mobile and is probably the safest if we do add another type. Do you think that'd be better?